src/python/WMCore/FwkJobReport/XMLParser.py

```diff
@@ -233,7 +233,7 @@ def runHandler():
                           if "ID" in lumi.attrs]
 
                 runInfo = Run(runNumber = runId)
-                runInfo.lumis.extend(lumis)
+                runInfo._lumis.extend(lumis)
```

Comment:
Yes, that a problem. The property allows to access internal member data, but it
does not mean that it allows applying methods of that member data type. So our
internal member data is self._lumis which is a list type. The lumis property
points to it, but does not allow to use list APIs (e.g. append, extend, etc).
What should be properly done is the following. We need to add method
extendLumis to Run class and use it instead of accessing internal data member.
But I don't know how many places in WMCore "relies" on direct internal member
data access and moreover using its type APIs.

On  0, Eric Vaandering <notifications@github.com> wrote:
> > @@ -233,7 +233,7 @@ def runHandler():
> >                            if "ID" in lumi.attrs]
> >  
> >                  runInfo = Run(runNumber = runId)
> > -                runInfo.lumis.extend(lumis)
> > +                runInfo._lumis.extend(lumis)
> 
> Help me understand this. I would have thought that with the @property decorator would allow you to just lumis?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6292/files#r40930193